troposphere/route53.py

```diff
@@ -7,6 +7,18 @@
 from .validators import integer, positive_integer, network_port, boolean
 
 
+VALID_RULETYPES = ('SYSTEM', 'FORWARD')
+
+
+def validate_ruletype(ruletype):
+    """Validate database Engine for DBInstance """
```

Comment:
This comment should be updated since it appears to be a copy/paste.

Reply:
Fixed, thanks for your review